README.md

```diff
@@ -18,62 +18,49 @@ are merged together by [Core Template Studio](https://github.com/Microsoft/CoreT
-
-- [Application Arquitecture](docs/contributing/application-architecture.md)
-- [Getting started with the codebase](/docs/contributing/getting-started-developers.md)
-- [Templates](./docs/contributing/templates.md)
-- [Accessibility](./docs/contributing/accessibility.md)
 
 ## SHA256
 
-Visual Studio Code extensions are currently not signed. WebTS provides the Secure Hash Algorithm (SHA) of each version we publish. You can find the current extensions sha256 value in the following files:
+Visual Studio Code extensions are currently not signed. *WebTS* provides the Secure Hash Algorithm (SHA) of each version we publish. You can find the current extensions sha256 value in the following files:
 
 - [sha256-release.md](sha256-release.md)
 
-To verify the integrity of the package, download the vsix from the Marketplace, calculate the hash of the package using the powershell command ‘Get-FileHash’ or any other tool that allows you to calculate the packages hash, and compare both.
+To verify the integrity of the package, download the `vsix` from the Marketplace, calculate the hash of the package using the powershell command `Get-FileHash` or any other tool that allows you to calculate the packages hash, and compare both.
 
 ## Instructions To Use
 
 ### Create a new project
 
 - Open **VSCode**
 - Press `Ctrl + Shift ⇧ + P` in Windows/Linux or `Command ⌘ + Shift ⇧ + P` in Mac to open VSCode's extension launcher
-- Type/Select `Web Template Studio: Launch` and press `Enter` to launch the extension
+- Type/Select `Web Template Studio: Create Web App` for a full-stack web application or `Web Template Studio: Create React Native App` for a React Native one, and press `Enter` to launch the extension
 
 ### Deploy the generated project
 
 - Open **VSCode**
 - Open the generated project
 - Press `Ctrl + Shift ⇧ + P` in Windows/Linux or `Command ⌘ + Shift ⇧ + P` in Mac to open VSCode's extension launcher
-- Type/Select `Web Template Studio: Deploy App` and press `Enter` to begin the deployment
+- Type/Select `Web Template Studio: Deploy Web App` and press `Enter` to begin the deployment
+**Note**: Just available for web applications.
 
 ## Example Scenario
 
-I need a full-stack web app that is deployed on Azure App Service Web App.
+### 1. Scenario 1
+ I need a full-stack web app that is deployed on Azure App Service Web App.
 
 <img alt="App with Azure Services" src="./docs/resources/readme-app-azure-screenshot.png" width="95%" />
 
+### 2. Scenario 2
+I want to create a react native app that works for Android and Windows.
+
+<img alt="App with Azure Services" src="./docs/resources/webts-dark-theme-react-native.png" width="95%" />
+
+Android App             |  Windows App
+:-------------------------:|:-------------------------:
+<img alt="Basic React Native app for Android" src="./docs/resources/react-native-sample-android-app.png" width="550" />  |  <img alt="Basic React Native app for Windows" src="./docs/resources/react-native-sample-windows-app.png" />
+
```

Comment:
Not sure if this should be here or on the description of the settings/master-detail page. 
We should include screenshots for iOS too.

Reply:
Will move this screenshots without table-format to page info